### ISSUE TITLE
Lose Christmas Gear When Changing Classes?

### ISSUE THREAD

**Issue by chadschultz** [HabitRPG/habitica#2307]
Seeing the variety of Christmas-themed gear for each class, I bought the gear, bought some gems, changed my class, bought some gems, changed my class... buying the special Christmas-themed gear for each of the four classes. I already had all the Warrior and Mage gear, but had no Rogue or Healer gear before. When I finished, I checked my inventory... and was quite surprised to only see the Christmas gear for my current class! Shouldn't it carry over on class changes, like other gear?

e7c9f489-4995-4448-b585-937712834bd8

**Comment by lefnire** [comment]
Showing you own all the gear (in database), sure it's not under your
"Special" heading?
https://www.evernote.com/shard/s17/sh/5aa46b2c-433a-4e63-b018-b6c6d8b46668/7c8271307ea0db7676e0ee99b7871638

**Comment by chadschultz** [comment]
Wow, I feel like an idiot-you are absolutely right. Never mind then, sorry!

**Comment by lefnire** [comment]
no worries, I'm trying to figure out how to give them class bonuses, and if I can solve that they'll show under their class's header. but this for now.

**Comment by chadschultz** [comment]
I used the Orb of Rebirth and was disappointed not to see the gear I had loaded up on. I had made sure to get it for every class before I rebooted the character. I assumed it would show up again once I reached a high enough level that I could choose a class... or once I had purchased some regular ninja equipment... or once I had purchased ALL the ninja equipment... no luck. I'm a level 19 ninja now. I've purchased all the ninja gear. However, when I look in my inventory, the only special items I see are ones I've purchased since AFTER the rebirth. Is this a bug? I believe special equipment is supposed to persist even after the Orb of Rebirth, right?
